pkg/cedar/intake/models/easi_intake.go

```diff
@@ -6,50 +6,50 @@ package models
-	ProjectName                 string `json:"projectName"`
-	RejectionReason             string `json:"rejectionReason"`
-	RequestType                 string `json:"requestType"`
-	Requester                   string `json:"requester"`
-	RequesterEmailAddress       string `json:"requesterEmailAddress"`
-	Solution                    string `json:"solution"`
-	Status                      string `json:"status"`
-	SubmittedAt                 string `json:"submittedAt"`
-	TrbCollaborator             string `json:"trbCollaborator"`
-	TrbCollaboratorName         string `json:"trbCollaboratorName"`
-	UserEUA                     string `json:"userEUA"`
+	AdminLead                   string `json:"adminLead" jsonschema:"description=Government Admin responsible for handling request,example=John Doe"`
+	ArchivedAt                  string `json:"archivedAt" jsonschema:"description=Timestamp of when request was archived,example=2022-02-17 14:34:43.405934+00"`
+	BusinessNeed                string `json:"businessNeed" jsonschema:"description=Business Need (problem space, benefits, deadlines, etc.) for the effort detailed in this request,example="`
+	BusinessOwner               string `json:"businessOwner" jsonschema:"description=Person who owns a line of business related to this request,example=John Doe"`
+	BusinessOwnerComponent      string `json:"businessOwnerComponent" jsonschema:"description=Component of the Business Owner,example=OIT"`
+	Component                   string `json:"component" jsonschema:"description=Component of the person who submitted this request,example=OIT"`
+	ContractEndDate             string `json:"contractEndDate" jsonschema:"description=The contract's end date,example=2026-10-20"`
+	ContractStartDate           string `json:"contractStartDate" jsonschema:"description=The contract's start date,example=2022-10-20"`
+	ContractVehicle             string `json:"contractVehicle" jsonschema:"description=Contract vehicle for this effort,example=8(a)"`
+	Contractor                  string `json:"contractor" jsonschema:"description=Contractor who will perform the work detailed in this request,example=Oddball"`
+	CostIncrease                string `json:"costIncrease" jsonschema:"description=Is there a cost increase associated with this request,example=YES"`
+	CostIncreaseAmount          string `json:"costIncreaseAmount" jsonschema:"description=How much is the cost increase,example=Over two million dollars"`
+	DecidedAt                   string `json:"decidedAt" jsonschema:"description=Timestamp of when decision was reached,example=2022-02-17 14:34:43.405934+00"`
+	DecisionNextSteps           string `json:"decisionNextSteps" jsonschema:"description=Steps that the business owner should take after receiving their decision,example=Go get a contract"`
+	EaCollaborator              string `json:"eaCollaborator" jsonschema:"description=N/A,example=N/A"` // TODO: doesn't seem like this is ever populated, remove?
```

Comment:
I think this EaCollaborator (along with Isso, OitSecurityCollaborator, and TrbCollaborator) was/is supposed to be used for checking whether there was a collaborator and then the collaborator's names are stored in the corresponding name fields. I don't think these fields are ever filled out in the DB.

We probably just want to remove these but didn't want to do that in this PR.

Reply:
https://jiraent.cms.gov/browse/EASI-1695